pkg/extensions/search/cve/trivy/scanner.go

```diff
@@ -265,7 +266,17 @@ func (scanner Scanner) ScanImage(image string) (map[string]cvemodel.CVE, error)
 
 	scanner.dbLock.Lock()
```

Comment:
Why no just add a defer unlock right here? and remove the other lock/unlock pairs below

Reply:
I thought other scanning goroutines should not be blocked while waiting for unrelated reports to be processed.

Reply:
I no longer modify this function.